packages/angular/src/sdk.ts

```diff
@@ -1,25 +1,52 @@
 import { VERSION } from '@angular/core';
 import type { BrowserOptions } from '@sentry/browser';
-import { getDefaultIntegrations, init as browserInit, setContext } from '@sentry/browser';
-import { applySdkMetadata } from '@sentry/core';
+import {
+  breadcrumbsIntegration,
+  globalHandlersIntegration,
+  httpContextIntegration,
+  linkedErrorsIntegration,
+} from '@sentry/browser';
+import { init as browserInit, setContext } from '@sentry/browser';
+import {
+  applySdkMetadata,
+  dedupeIntegration,
+  functionToStringIntegration,
+  inboundFiltersIntegration,
+} from '@sentry/core';
+import type { Integration } from '@sentry/types';
 import { logger } from '@sentry/utils';
 
 import { IS_DEBUG_BUILD } from './flags';
 
+/**
+ * Get the default integrations for the Angular SDK.
+ */
+export function getDefaultIntegrations(): Integration[] {
```

Comment:
I realize this is already merged but @AbhiPrasad  is there a reason we don't just call `getDefaultIntegrations` from browser and filter out `BrowserApiErrors`? This way we'd always be in sync with browser otherwise and we don't need the reminder comments.

Reply:
Ahh sorry blacked out there for a second. To answer my own question, it's because of bundle size; `BrowserApiErrors` would otherwise still be in the bundle although it's not used.